pkg/chunked/bloom_filter.go

```diff
@@ -28,7 +31,7 @@ func newBloomFilterFromArray(bitArray []uint64, k uint32) *bloomFilter {
 
 func (bf *bloomFilter) hashFn(item []byte, seed uint32) (uint64, uint64) {
 	if len(item) == 0 {
-		return 0, 0
+		return 0, 1
```

Comment:
Is this correct? If we have a 1-bit filter, it’s a bit `hash == 0`, i.e. `(0 / 64, 0 % 64) == (0, 0)`, isn’t it?

Reply:
… never mind, the value returned is a mask, not a bit index.